JSDemos/Demos/Toolbar/Overview/description.md

```diff
@@ -1 +1,35 @@
-The Toolbar is&nbsp;a&nbsp;UI component containing items that usually manage screen content. Those items can be&nbsp;plain text or&nbsp;widgets.
\ No newline at end of file
+The Toolbar contains items that manage the page content: other components and screen view. In this demo, the Toolbar manages the [List](/Documentation/ApiReference/UI_Components/dxList/).
+
+## Configure Toolbar Items
+
+You can store Toolbar items in the [items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array or specify the [dataSource](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#dataSource). A Toolbar item may be plain text or a UI component. You should specify the [text](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#text) or the [widget](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#widget) property depending on the item. If the item is a UI component, declare its [options](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#options).
+
+## Specify Item Location
+
+You can set the **dataSource** with **location** fields or specify the [location](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#location) property for each item. The **location** value can be on of the following:
+
+- *"center"*  
+ Places the item in the center of the toolbar.
+
+- *"before"*  
+ Places the item before the central element(s).
+
+- *"after"*  
+ Places the item after the central element(s).
+
+Additionally, the Toolbar can render its items in the overflow menu. Specify the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property for each item with one of the following values:
+
+- *"always"*  
+ Always places the item in the overflow menu.
+
+- *"never"*  
+ Places the item outside of the overflow menu.
+
+- *"auto"*  
+ Places the item outside of the overflow menu. If the width of the Toolbar is not enough to store all the items, the Toolbar renders this item in the overflow menu.
```

Comment:
`If the width of the Toolbar is not enough to store all the items, the Toolbar renders this item in the overflow menu.`

I'd like to update "not enough". Can we change this sentence to one of the following?

`If the width of the Toolbar cannot store all items, the Toolbar renders this item in the overflow menu.`

or

`If all items cannot fit within the width of the Toolbar, it renders this item in the overflow menu.`

Reply:
I'm going to change it to the last one.